index.html

```diff
@@ -0,0 +1,670 @@
+      </p>
+      <p>
+        Almost as quickly, this pulse would subside, relaxing into ordinary particles and radiation. All of normal cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars billions of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to say, “but the universe might be the ultimate free lunch.”
+      </p>
+      <p>
+      Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision suggest that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a concept known as the multiverse.
+    </p>
+    <p>
+      So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any deeper into the past.
+    </p>
+    <p>
+      We might never know what happened before inflation, at the very beginning, because inflation erases everything that came before it. All the chaos and randomness of the primordial moment are swept away, forever out of our view.
+    </p>
+    <p>
+    “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of the team, “you wind up at inflation.”
+    </p>
+    <section class="article-footer">
+      <small>
+        A version of this article appears in print on March 18, 2014, Section A, Page 1 of the New York edition with the headline: Space Ripples Reveal Big Bang’s Smoking Gun. Order Reprints | Today’s Paper | Subscribe
+      </small>
+      <br />
+      <div class="more-comments">
+        <button id="comments" type="button">Read 615 comments</button>
+        <div class="social-icons">
+          <i class="fa fa-facebook" aria-hidden="true"></i>
+          &ensp;
+          <i class="fa fa-twitter" aria-hidden="true"></i>
+          &ensp;
+          <i class="fa fa-envelope" aria-hidden="true"></i>
+          &ensp;
+          <i class="fa fa-share" aria-hidden="true"></i>
+        </div>
+      </div>
+    </section>
+    </section>
+    <section class="more-section">
+      <div class="grid-section">
+        <h4>More in space and astronomy</h4>
+        <div class="grid-container">
+          <div class="grid-item">
+            <img src="assets/images/16SOLARORBITER-loop-cover-threeByTwoSmallAt2X.jpg" alt="">
+            <small>Solar Orbiter/EUI Team(ESA & NASA)</small>
+            <a href="https://www.nytimes.com/2020/07/16/science/solar-orbiter-sun-images.html?action=click&block=more_in_recirc&impression_id=603315740&index=0&pgtype=Article&region=footer"><strong>Closest Pictures Ever Taken of Sun Show Tiny Campfire Flares</strong><span>July 17</span></a>
+          </div>
```

Comment:
## Links to other articles

It contains links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

- [ ] Add `<a>` to whole element class `grid-item` not only the figcaption